### ISSUE TITLE
Get generated key from autoIncrement field

### ISSUE THREAD

**Issue by 8alery** [jakearchibald/idb#96]
Good afternoon!
I tried to get the value of autoIncrement field but add method on db object returns undefined. But new item is added, I can see it after I refresh page. I couldn't find information on it in docs so I'd like to ask how can I get this generated id? My code is like this: 

```
import { openDB } from 'idb';
const dbPromise = openDB('investment_db', 1, {
	upgrade(db) {
		const ordersStore = db.createObjectStore('orders', {
			keyPath: 'id',
			autoIncrement: true,
		});
		ordersStore.createIndex('purchaseDate', 'purchaseDate');
	},
});
export default {
        async addOrder(order){
		return (await dbPromise).add('orders', order);
        },
        async getOrders(){
		return (await dbPromise).getAllFromIndex('orders', 'purchaseDate').then((items) => {
			return items;
		})
	 }
}
```

**Comment by jakearchibald** [comment]
This is a bug. Well spotted.

As a workaround, create a transaction:

```js
// Instead of 
(await dbPromise).add('orders', order);
// Do this:
const tx = (await dbPromise).transaction('orders', 'readwrite');
const id = await tx.store.add(order);
await tx.done;
```

**Comment by jakearchibald** [comment]
Fixed in 2f24955fb63787faa118450092e02bc72d0929cd. Published in 4.0.1.
